src/App.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Home from './pages/Home';
+import Search from './pages/Search';
+
 
 function App() {
   return (
-    <div>
-      App Placeholder
-    </div>
+    <Router>
+      <Switch>
+        <Route exact path="/" component={Home} />
```

Comment:
This is completely right :+1: Instead of using the `component` prop you could also render the page like a normal child.

```
<Route>
  <Home />
</Route>
```

That's the way it's described in the docs. Not sure if there is an advantage though.

Another thing you could do is adding a fallback. Either a 404 page or a redirect to `/` as last child in `Switch`.

```
<Redirect to="/" />
```
or

```
<Route>404 content</Route>
```